### PR TITLE
Adding new downtime warning component

### PR DESCRIPTION
**Pull request by PeteWilliams**
Have made a start on the downtime warning component.
First time I've added a new component, so can you check all looks right please @adamsilver ?
In the end, I don't think it really needs a video - given that it's static, and I think words would do a better job of explaining that it's only shown on the first screen.
Content hasn't been reviewed yet, so I'll need to find a volunteer.…

### CODE REVIEW

File: app/views/patterns/be-informed-of-downtime/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+{{dsExample({
+  section: 'patterns',
+  name: 'be-informed-of-downtime',
+  example: 'information',
+  height: 175
+})}}
+
+## When to use this pattern
+
+Use this component when you know in advance that your service will be unavailable.
```
Comment (adamsilver) [inline app/views/patterns/be-informed-of-downtime/README.md]: Change “component” to “pattern”

**Comment by PeteWilliams** [comment]
Good point, have made those changes.

**Comment by PeteWilliams** [comment]
@adamsilver - Have made those changes now.

**Comment by adamsilver** [comment]
Nice, merging.

**Pull request merged**